common-api/src/main/resources/common_api.yaml

```diff
@@ -205,6 +205,9 @@ definitions:
       id:
         type: string
         description: the Firecloud id of the workspace
+      category:
+        type: string
```

Comment:
Can you add an "enum: [PHENOTYPE_LIBRARY, TUTORIAL_WORKSPACES]" here, or defined separately as a top-level enum?